<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Isobel Beasley &amp; Yasmine McDonough" />


<title>Sex biased mutation rate</title>

<script src="site_libs/header-attrs-2.9/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/core-js-2.5.3/shim.min.js"></script>
<script src="site_libs/react-17.0.0/react.min.js"></script>
<script src="site_libs/react-17.0.0/react-dom.min.js"></script>
<script src="site_libs/reactwidget-1.0.0/react-tools.js"></script>
<script src="site_libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="site_libs/reactable-binding-0.2.3/reactable.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">unguarded_x_journal_club</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/IJbeasley/unguarded_x_journal_club">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Sex biased mutation rate</h1>
<h4 class="author">Isobel Beasley &amp; Yasmine McDonough</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-02-06
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>unguarded_x_journal_club/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20211208code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20211208)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20211208code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20211208)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomIJbeasleyunguardedxjournalclubtree3751ab00dbb8a15d62e18dc05f1b8072b1d857c9targetblank3751ab0a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/IJbeasley/unguarded_x_journal_club/tree/3751ab00dbb8a15d62e18dc05f1b8072b1d857c9" target="_blank">3751ab0</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomIJbeasleyunguardedxjournalclubtree3751ab00dbb8a15d62e18dc05f1b8072b1d857c9targetblank3751ab0a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/IJbeasley/unguarded_x_journal_club/tree/3751ab00dbb8a15d62e18dc05f1b8072b1d857c9" target="_blank">3751ab0</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    analysis/figure/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/sex_biased_mutation_rate.Rmd</code>) and HTML (<code>docs/sex_biased_mutation_rate.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/IJbeasley/unguarded_x_journal_club/blob/3751ab00dbb8a15d62e18dc05f1b8072b1d857c9/analysis/sex_biased_mutation_rate.Rmd" target="_blank">3751ab0</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2022-02-06
</td>
<td>
Update workflowr page for sex_biased_mutation_rate
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/IJbeasley/unguarded_x_journal_club/1999cf48b11195375c7eb974f3ddd366e837a2d3/docs/sex_biased_mutation_rate.html" target="_blank">1999cf4</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/IJbeasley/unguarded_x_journal_club/blob/5086eac8adbb63241fe9b3050f1ccac32aa908b1/analysis/sex_biased_mutation_rate.Rmd" target="_blank">5086eac</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-10
</td>
<td>
Include information on the data collated for sex-biased mutation rate
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/IJbeasley/unguarded_x_journal_club/93f651a8f1fa8ec3f158d6e39104632a3ac6e50a/docs/sex_biased_mutation_rate.html" target="_blank">93f651a</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/IJbeasley/unguarded_x_journal_club/blob/5066bc3603932c0c726f69111bbe476d0660775c/analysis/sex_biased_mutation_rate.Rmd" target="_blank">5066bc3</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-09
</td>
<td>
Update sex biased mut rate dist by method used calculate
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/IJbeasley/unguarded_x_journal_club/a3e012c7598dacc697eaa3ea74ea4509e1935aa1/docs/sex_biased_mutation_rate.html" target="_blank">a3e012c</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/IJbeasley/unguarded_x_journal_club/blob/af37ec4651e5ccfba01105d4a7f34705e626152e/analysis/sex_biased_mutation_rate.Rmd" target="_blank">af37ec4</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-09
</td>
<td>
Include dist of sex bias est by taxo class
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/IJbeasley/unguarded_x_journal_club/31b9e9f0e0a8bc240273392f2566dd65e5b12f1e/docs/sex_biased_mutation_rate.html" target="_blank">31b9e9f</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/IJbeasley/unguarded_x_journal_club/blob/3d510d6591fc35cef8002fa2a0e7c943ae8f0825/analysis/sex_biased_mutation_rate.Rmd" target="_blank">3d510d6</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-08
</td>
<td>
Add inital distribution graph of sex biased mut rate
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/IJbeasley/unguarded_x_journal_club/a1238f3b60e7c2d398e553778deb651111e4212c/docs/sex_biased_mutation_rate.html" target="_blank">a1238f3</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/IJbeasley/unguarded_x_journal_club/blob/51d2d42a498f5e8da68d9a8b56a95cb52a06e90b/analysis/sex_biased_mutation_rate.Rmd" target="_blank">51d2d42</a>
</td>
<td>
Isobel Beasley
</td>
<td>
2021-12-08
</td>
<td>
Publish inital webpages project
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p><strong>Introduction</strong> This workflowr page contains investigations into Additionally - it illustrates how we produced Figure 1 for the paper.</p>
<div id="set-up" class="section level1">
<h1>Set up</h1>
<pre class="r"><code>library(ggplot2)
suppressMessages(library(dplyr))
library(reactable)</code></pre>
<pre><code>Warning: package &#39;reactable&#39; was built under R version 4.1.2</code></pre>
<pre class="r"><code>library(tidyr)

# only look at public sheets - so no authentication is needed
googlesheets4::gs4_deauth()

# read in sex bias mutation data.frame
sex_bias_mut_df = googlesheets4::read_sheet(&quot;https://docs.google.com/spreadsheets/d/10ZVgeGlDpEnPG-4zLLrSBQjF7p2_9hzzIMPuIRuvogg/edit#gid=537547888&quot;, range = &quot;Sex-bias estimates&quot;)</code></pre>
<pre><code>v Reading from &quot;Sex_bias_mutation_rate&quot;.</code></pre>
<pre><code>v Range &#39;&#39;Sex-bias estimates&#39;&#39;.</code></pre>
<pre class="r"><code># what does the sex bias in mutation rate estimate look like: 
head(sex_bias_mut_df) %&gt;% reactable()</code></pre>
<div id="htmlwidget-0bc58612b86b389075fb" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-0bc58612b86b389075fb">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"authors":["Shimmin, Chang and Li","Chang et al.","Chang and Li","Chang, Hewett-Emmett and Li","Ellegren and Fridolfsson","Huang et al."],"year":[1993,1994,1995,1996,1997,1997],"taxonomic_class":["Mammalia","Mammalia","Mammalia","Mammalia","Aves","Mammalia"],"animal_name":["Primates (Human, Orangutan, Baboons, Squirrel Monkey)","Rodents (Mouse;Rat)","Rodents (Mouse;Rat)","Primates (Human, Orangutan, Baboons, Squirrel Monkey)","Birds (Collared flycatcher;Yellowhammer)","Primates (Human, Orangutan, Squirrel Monkey)"],"scientific_name":["Homo sapiens;Pongo pygmaeus;Papio\ncynocephalus;Saimiri boliviensis","Mus musculus;Rattus norvegicus","Mus musculus;Rattus norvegicus","Homo sapiens;Pongo pygmaeus;Papio\ncynocephalus;Saimiri boliviensis","Ficedula albicollis;Emberiza citrinella","Homo sapiens;Pongo pygmaeus;Saimiri boliviensis"],"alpha_point_estimate":[6.26,1,2,4.2,3.9,5.14],"alpha_estimate_lower":[2.63,1.8,1,2.2,1.12,2.42],"alpha_estimate_upper":[32.4,3.2,3.9,10,6.68,16.6],"method_used_to_calculate_alpha":["Evolutionary method (Miyata et al. 1987, comparing X/Y)","Evolutionary method (Miyata et al. 1987, comparing X/Y)","Evolutionary method (Miyata et al. 1987, comparing X/Y)","Evolutionary method (Miyata et al. 1987, comparing X/Y)","Evolutionary method (Miyata et al. 1987; comparing Z/W)","Evolutionary method (Miyata et al. 1987, comparing X/Y)"],"upper_lower_meaning":["Lower and upper estimates are the 95% confidence interval for alpha","Lower and upper estimates are the 95% confidence interval for alpha","Lower and upper estimates are the 95% confidence interval for alpha","Lower and upper estimates are the 95% confidence interval for alpha","Lower and upper estimates are the 95% confidence interval for alpha","Lower and upper estimates are the 95% confidence interval for alpha"],"loci_type_for_estimate":["Intron","Intron","Intron","Intron","Exon","Intron"],"number_of_loci_for_estimate":["Single gene pair","Single gene pair","Single gene pair","Single gene pair","Single gene pair","Single gene pair"],"mutation_type":[null,null,null,null,null,null],"other_notes":["Gene pair: ZFY/ZFX","Gene pair: ZFY/ZFX","Gene pair: Ube 1X/Ube 1Y","Gene pair: SMCY/SMCX","Gene pair: CHDZ / CHDW","Gene pair: AMGX/AMGY"]},"columns":[{"accessor":"authors","name":"authors","type":"character"},{"accessor":"year","name":"year","type":"numeric"},{"accessor":"taxonomic_class","name":"taxonomic_class","type":"character"},{"accessor":"animal_name","name":"animal_name","type":"character"},{"accessor":"scientific_name","name":"scientific_name","type":"character"},{"accessor":"alpha_point_estimate","name":"alpha_point_estimate","type":"numeric"},{"accessor":"alpha_estimate_lower","name":"alpha_estimate_lower","type":"numeric"},{"accessor":"alpha_estimate_upper","name":"alpha_estimate_upper","type":"numeric"},{"accessor":"method_used_to_calculate_alpha","name":"method_used_to_calculate_alpha","type":"character"},{"accessor":"upper_lower_meaning","name":"upper_lower_meaning","type":"character"},{"accessor":"loci_type_for_estimate","name":"loci_type_for_estimate","type":"character"},{"accessor":"number_of_loci_for_estimate","name":"number_of_loci_for_estimate","type":"character"},{"accessor":"mutation_type","name":"mutation_type","type":"character"},{"accessor":"other_notes","name":"other_notes","type":"character"}],"defaultPageSize":10,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"c7a55accab30ba4d2a8968dccbe0b3cd","key":"c7a55accab30ba4d2a8968dccbe0b3cd"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code># Set up custom theme for ggplots
custom_theme &lt;-
  list(
    theme_bw() +
      theme(
        panel.border = element_blank(),
        axis.line = element_line(),
        text = element_text(size = 16),
        legend.position = &quot;bottom&quot;,
        strip.background = element_blank(),
        axis.text.x = element_text(angle = 90, hjust = 1, vjust = 0.5)
      )
  )</code></pre>
</div>
<div id="what-data-have-we-collected-collated" class="section level1">
<h1>What data have we collected / collated?</h1>
<div id="number-of-unique-studies-included" class="section level2">
<h2>Number of unique studies included</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  select(authors,year) %&gt;% 
  distinct() %&gt;% 
  nrow()</code></pre>
<pre><code>[1] 46</code></pre>
</div>
<div id="when-where-these-studies-published" class="section level2">
<h2>When where these studies published?</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;%   
  select(authors,year) %&gt;% 
  distinct() %&gt;% 
  ggplot(aes(x=year)) + 
  geom_histogram() + 
  custom_theme  + 
  labs(x = &quot;Year published*&quot;,
       y = &quot;Number&quot;,
       title = &quot;What year were our collated studies published?&quot;)</code></pre>
<pre><code>`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/sex_biased_mutation_rate.Rmd/when_studies_published-1.png" width="1152" style="display: block; margin: auto;" /></p>
<ul>
<li>Or preprinted, in the case of included preprints</li>
</ul>
</div>
<div id="how-many-distinct-estimates-per-year" class="section level2">
<h2>How many distinct estimates per year?</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;%   
  select(authors,year,scientific_name) %&gt;% 
  distinct() %&gt;% 
  ggplot(aes(x=year)) + 
  geom_histogram() + 
  custom_theme  + 
  labs(x = &quot;Year published*&quot;,
       y = &quot;Number&quot;,
       title = &quot;How many unique estimtates (study, and organism specific) per year?&quot;)</code></pre>
<pre><code>`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/sex_biased_mutation_rate.Rmd/n_samples_per_year-1.png" width="1152" style="display: block; margin: auto;" /></p>
<ul>
<li>Or preprinted, in the case of included preprints</li>
</ul>
</div>
<div id="number-of-estimates-of-sex-biased-mutation-rate-per-taxonomic-class" class="section level2">
<h2>Number of estimates of sex-biased mutation rate per taxonomic class</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  group_by(taxonomic_class) %&gt;% 
  summarise(n = n()) %&gt;% 
  reactable()</code></pre>
<div id="htmlwidget-a71b15f820af623e53da" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-a71b15f820af623e53da">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"taxonomic_class":["Actinopterygii","Amphibia","Arachnida","Aves","Insecta","Mammalia","Reptilia"],"n":[1,2,2,74,3,71,5]},"columns":[{"accessor":"taxonomic_class","name":"taxonomic_class","type":"character"},{"accessor":"n","name":"n","type":"numeric"}],"defaultPageSize":10,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"b37d009d332399f309daa81edb54b66f","key":"b37d009d332399f309daa81edb54b66f"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="number-of-estimates-per-method-used-to-calculate-the-sex-biased-ratio-in-mutation-rate" class="section level2">
<h2>Number of estimates per method used to calculate the sex-biased ratio in mutation rate</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  group_by(method_used_to_calculate_alpha) %&gt;% 
  summarise(n = n()) %&gt;% 
  reactable(defaultPageSize = 50)</code></pre>
<div id="htmlwidget-953d50596abf826ad1b4" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-953d50596abf826ad1b4">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"method_used_to_calculate_alpha":["Direct estimate (trios)","Evolutionary method (Miyata et al. 1987, comparing X/Y)","Evolutionary method (Miyata et al. 1987; comparing W/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing X/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing X/Y)","Evolutionary method (Miyata et al. 1987; comparing Y/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing Z/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing Z/W)"],"n":[14,5,1,49,9,6,58,16]},"columns":[{"accessor":"method_used_to_calculate_alpha","name":"method_used_to_calculate_alpha","type":"character"},{"accessor":"n","name":"n","type":"numeric"}],"defaultPageSize":50,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"6d4b1574f21840c6957f26e7bd331f36","key":"6d4b1574f21840c6957f26e7bd331f36"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="number-of-estimates-per-loci-type-used-to-calculate-the-sex-biased-ratio-in-mutation-rate" class="section level2">
<h2>Number of estimates per loci type used to calculate the sex-biased ratio in mutation rate</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  group_by(loci_type_for_estimate) %&gt;% 
  summarise(n = n()) %&gt;% 
  reactable(defaultPageSize = 50)</code></pre>
<div id="htmlwidget-dfb6bc2cdefdbd9c15ee" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-dfb6bc2cdefdbd9c15ee">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"loci_type_for_estimate":["Ancestral repeat","Ancestral repeats","Exon","Intron","Microsattelite","Non-coding","Noncoding sequence","Short tandem repeat (STR)","Synomous site","Synonymous site","Whole genome",null],"n":[1,2,1,76,2,32,1,1,1,12,1,28]},"columns":[{"accessor":"loci_type_for_estimate","name":"loci_type_for_estimate","type":"character"},{"accessor":"n","name":"n","type":"numeric"}],"defaultPageSize":50,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"ebdd64bdf67086753b764a970f5264da","key":"ebdd64bdf67086753b764a970f5264da"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
<pre class="r"><code># Number of unique studies

sex_bias_mut_df %&gt;% 
  group_by(authors,year) %&gt;% 
  sample_n(1) %&gt;% 
  ungroup() %&gt;% 
  group_by(taxonomic_class,loci_type_for_estimate) %&gt;% 
  summarise(n = n()) %&gt;% 
  reactable(defaultPageSize = 50)</code></pre>
<pre><code>`summarise()` has grouped output by &#39;taxonomic_class&#39;. You can override using the `.groups` argument.</code></pre>
<div id="htmlwidget-780cb9f563e2acfaa2c2" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-780cb9f563e2acfaa2c2">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"taxonomic_class":["Actinopterygii","Amphibia","Arachnida","Aves","Aves","Aves","Aves","Aves","Aves","Insecta","Insecta","Mammalia","Mammalia","Mammalia","Mammalia","Mammalia","Mammalia","Mammalia","Mammalia","Reptilia"],"loci_type_for_estimate":["Intron",null,"Synonymous site","Ancestral repeat","Exon","Intron","Microsattelite","Synonymous site",null,"Synomous site",null,"Ancestral repeats","Intron","Microsattelite","Non-coding","Noncoding sequence","Short tandem repeat (STR)","Whole genome",null,null],"n":[1,1,1,1,1,5,1,1,2,1,1,1,8,1,1,1,1,1,13,3]},"columns":[{"accessor":"taxonomic_class","name":"taxonomic_class","type":"character"},{"accessor":"loci_type_for_estimate","name":"loci_type_for_estimate","type":"character"},{"accessor":"n","name":"n","type":"numeric"}],"defaultPageSize":50,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"2ff56732c8183689d422bde3864f32a8","key":"2ff56732c8183689d422bde3864f32a8"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="number-of-estimates-by-both-taxonomic-class-and-method-used-to-calculate" class="section level2">
<h2>Number of estimates by both taxonomic class and method used to calculate</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  group_by(taxonomic_class,method_used_to_calculate_alpha) %&gt;% 
  summarise(n = n()) %&gt;% 
  reactable(defaultPageSize = 50)</code></pre>
<pre><code>`summarise()` has grouped output by &#39;taxonomic_class&#39;. You can override using the `.groups` argument.</code></pre>
<div id="htmlwidget-de5fccbccc4019eddf97" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-de5fccbccc4019eddf97">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"taxonomic_class":["Actinopterygii","Amphibia","Amphibia","Arachnida","Aves","Aves","Aves","Aves","Insecta","Mammalia","Mammalia","Mammalia","Mammalia","Mammalia","Reptilia","Reptilia","Reptilia","Reptilia"],"method_used_to_calculate_alpha":["Evolutionary method (Miyata et al. 1987; comparing Y/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing X/Y)","Evolutionary method (Miyata et al. 1987; comparing Z/W)","Evolutionary method (Miyata et al. 1987; comparing X/Autosomes)","Direct estimate (trios)","Evolutionary method (Miyata et al. 1987; comparing W/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing Z/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing Z/W)","Evolutionary method (Miyata et al. 1987; comparing X/Autosomes)","Direct estimate (trios)","Evolutionary method (Miyata et al. 1987, comparing X/Y)","Evolutionary method (Miyata et al. 1987; comparing X/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing X/Y)","Evolutionary method (Miyata et al. 1987; comparing Y/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing X/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing X/Y)","Evolutionary method (Miyata et al. 1987; comparing Y/Autosomes)","Evolutionary method (Miyata et al. 1987; comparing Z/Autosomes)"],"n":[1,1,1,2,2,1,56,15,3,12,5,43,7,4,1,1,1,2]},"columns":[{"accessor":"taxonomic_class","name":"taxonomic_class","type":"character"},{"accessor":"method_used_to_calculate_alpha","name":"method_used_to_calculate_alpha","type":"character"},{"accessor":"n","name":"n","type":"numeric"}],"defaultPageSize":50,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"9b9e048f560c62e14a5531764c08fd8d","key":"9b9e048f560c62e14a5531764c08fd8d"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="distribution-of-sex-biased-mutation-estimates" class="section level1">
<h1>Distribution of sex-biased mutation estimates</h1>
<div id="across-all-taxonomic-classes" class="section level2">
<h2>Across all taxonomic classes</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  ggplot(aes(x=alpha_point_estimate)) + 
  geom_histogram(bins = 50) + 
  labs(title = &quot;Distribution of estimated sex-biased mutation rate&quot;,
        x = &quot;Point estimate of the male-biased mutation rate (alpha)&quot;,
       y= &quot;Count&quot;) + 
  custom_theme + 
  geom_vline(xintercept = 1, col = &quot;red&quot;, linetype = 2, size = 2) + 
  annotate(
  &quot;text&quot;,
  x = 3,
  y = 35,
  size = 9,
  hjust = 0,
  col = &quot;red&quot;, 
  label = &quot;Line of equal mutation rate across the sexes&quot;
)</code></pre>
<p><img src="figure/sex_biased_mutation_rate.Rmd/dist_sex_est-1.png" width="1152" style="display: block; margin: auto;" /></p>
<pre class="r"><code>summary(sex_bias_mut_df$alpha_point_estimate)</code></pre>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
 -0.480   1.992   2.365   2.873   3.337  20.090 </code></pre>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  ggplot(aes(x=alpha_point_estimate)) +
  geom_boxplot() + 
  custom_theme+ 
   labs(title = &quot;Distribution of estimated sex-biased mutation rate&quot;,
        x = &quot;Point estimate of the male-biased mutation rate (alpha)&quot;) </code></pre>
<p><img src="figure/sex_biased_mutation_rate.Rmd/dist_sex_est-2.png" width="1152" style="display: block; margin: auto;" /></p>
<div id="after-sampling-down-multiple-estimates-on-the-same-sample-i.e.-same-study-and-organism" class="section level3">
<h3>After sampling down multiple estimates on the same sample (i.e. same study, and organism)</h3>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  group_by(authors,year,scientific_name) %&gt;% 
  sample_n(1) %&gt;% 
  ggplot(aes(x=alpha_point_estimate)) + 
  geom_histogram(bins = 50) + 
  labs(title = &quot;Distribution of estimated sex-biased mutation rate&quot;,
        x = &quot;Point estimate of the male-biased mutation rate (alpha)&quot;,
       y= &quot;Number of estimates&quot;) + 
  custom_theme + 
  geom_vline(xintercept = 1, col = &quot;red&quot;, linetype = 2, size = 2) + 
  annotate(
  &quot;text&quot;,
  x = 3,
  y = 35,
  size = 9,
  hjust = 0,
  col = &quot;red&quot;, 
  label = &quot;Line of equal mutation rate across the sexes&quot;
)</code></pre>
<p><img src="figure/sex_biased_mutation_rate.Rmd/sampling_down_est_dist_est_in_mut_bias-1.png" width="1152" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="by-taxonomic-class" class="section level2">
<h2>By Taxonomic Class</h2>
<pre class="r"><code>df= sex_bias_mut_df %&gt;% 
  filter(alpha_point_estimate&lt;20) %&gt;% 
  ggplot(aes(x=alpha_point_estimate)) + 
  geom_histogram(bins = 25) + 
  labs(title = &quot;Distribution of sex-biased mutation rate estimtates&quot;,
        x = &quot;Point estimate of the male-biased mutation rate (alpha)&quot;,
       y= &quot;Number of estimates&quot;) + 
  custom_theme + 
  geom_vline(xintercept = 1, col = &quot;red&quot;, linetype = 2, size = 2) + 
  facet_wrap(~taxonomic_class)

n_by_taxa = sex_bias_mut_df %&gt;% 
            group_by(taxonomic_class) %&gt;% 
            summarise(n = n()) 

library(gridExtra)</code></pre>
<pre><code>
Attaching package: &#39;gridExtra&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:dplyr&#39;:

    combine</code></pre>
<pre class="r"><code>df_2 = sex_bias_mut_df %&gt;%   
  select(authors,year,scientific_name) %&gt;% 
  distinct() %&gt;% 
  ggplot(aes(x=year)) + 
  geom_histogram() + 
  custom_theme  + 
  labs(x = &quot;Year published*&quot;,
       y = &quot;Number of estimates&quot;,
       title = &quot;How many unique estimates per year?&quot;)


df_3 = sex_bias_mut_df %&gt;%   
  select(authors,year) %&gt;% 
  distinct() %&gt;% 
  ggplot(aes(x=year)) + 
  geom_histogram() + 
  custom_theme  + 
  labs(x = &quot;Year published*&quot;,
       y = &quot;Number of studies&quot;,
       title = &quot;What year were our collated studies published?&quot;)


library(patchwork)

(df) / (df_2 + df_3)</code></pre>
<pre><code>`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<pre><code>`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/sex_biased_mutation_rate.Rmd/sex_bias_by_taxo_class-1.png" width="1152" style="display: block; margin: auto;" /></p>
</div>
<div id="by-method-used-to-calculate-estimate-of-male-biased-mutation-rate" class="section level2">
<h2>By method used to calculate estimate of male-biased mutation rate</h2>
<pre class="r"><code>sex_bias_mut_df %&gt;% 
  ggplot(aes(x=alpha_point_estimate)) + 
  geom_histogram() + 
  custom_theme + 
  facet_wrap(~method_used_to_calculate_alpha) + 
  theme(text = element_text(size = 25)) +   
  labs(title = &quot;Distribution of estimated sex-biased mutation rate&quot;,
        x = &quot;Point estimate of the male-biased mutation rate (alpha)&quot;,
       y= &quot;Count&quot;)</code></pre>
<pre><code>`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.</code></pre>
<p><img src="figure/sex_biased_mutation_rate.Rmd/sex_bias_by_method-1.png" width="2880" style="display: block; margin: auto;" /></p>
</div>
<div id="confidence-intervals--" class="section level2">
<h2>Confidence intervals -</h2>
<pre class="r"><code>df= sex_bias_mut_df %&gt;% 
  filter(upper_lower_meaning == &quot;Lower and upper estimates are the 95% confidence interval for alpha&quot;|
         upper_lower_meaning == &quot;Lower and upper estimates are 2 standard errors from the mean (rough 95% confidence interval)&quot;) %&gt;% 
  mutate(se = (alpha_estimate_upper - alpha_point_estimate)/2) 


df %&gt;% 
  select(taxonomic_class,se) %&gt;% 
  mutate(row = row_number()) %&gt;% 
  pivot_wider(names_from = taxonomic_class,
              values_from = se,
              values_fn = list)</code></pre>
<pre><code># A tibble: 118 x 6
     row Mammalia  Aves      Amphibia Reptilia Insecta
   &lt;int&gt; &lt;list&gt;    &lt;list&gt;    &lt;list&gt;   &lt;list&gt;   &lt;list&gt; 
 1     1 &lt;dbl [1]&gt; &lt;NULL&gt;    &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
 2     2 &lt;dbl [1]&gt; &lt;NULL&gt;    &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
 3     3 &lt;dbl [1]&gt; &lt;NULL&gt;    &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
 4     4 &lt;dbl [1]&gt; &lt;NULL&gt;    &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
 5     5 &lt;NULL&gt;    &lt;dbl [1]&gt; &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
 6     6 &lt;dbl [1]&gt; &lt;NULL&gt;    &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
 7     7 &lt;NULL&gt;    &lt;dbl [1]&gt; &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
 8     8 &lt;NULL&gt;    &lt;dbl [1]&gt; &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
 9     9 &lt;dbl [1]&gt; &lt;NULL&gt;    &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
10    10 &lt;NULL&gt;    &lt;dbl [1]&gt; &lt;NULL&gt;   &lt;NULL&gt;   &lt;NULL&gt; 
# ... with 108 more rows</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252   
[3] LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] patchwork_1.1.1 gridExtra_2.3   tidyr_1.1.3     reactable_0.2.3
[5] dplyr_1.0.6     ggplot2_3.3.5   workflowr_1.6.2

loaded via a namespace (and not attached):
 [1] tidyselect_1.1.1    xfun_0.24           bslib_0.2.5.1      
 [4] rematch2_2.1.2      purrr_0.3.4         gargle_1.2.0       
 [7] colorspace_2.0-1    vctrs_0.3.8         generics_0.1.0     
[10] htmltools_0.5.1.1   yaml_2.2.1          utf8_1.2.1         
[13] rlang_0.4.11        jquerylib_0.1.4     later_1.2.0        
[16] pillar_1.6.1        reactR_0.4.4        glue_1.4.2         
[19] withr_2.4.2         DBI_1.1.1           lifecycle_1.0.0    
[22] stringr_1.4.0       munsell_0.5.0       gtable_0.3.0       
[25] cellranger_1.1.0    htmlwidgets_1.5.3   evaluate_0.14      
[28] labeling_0.4.2      knitr_1.33          crosstalk_1.1.1    
[31] httpuv_1.6.1        curl_4.3.1          fansi_0.4.2        
[34] highr_0.9           Rcpp_1.0.6          promises_1.2.0.1   
[37] scales_1.1.1        googlesheets4_1.0.0 jsonlite_1.7.2     
[40] farver_2.1.0        fs_1.5.0            digest_0.6.27      
[43] stringi_1.6.1       grid_4.1.0          rprojroot_2.0.2    
[46] cli_3.1.0           tools_4.1.0         magrittr_2.0.1     
[49] sass_0.4.0          tibble_3.1.2        crayon_1.4.1       
[52] whisker_0.4         pkgconfig_2.0.3     ellipsis_0.3.2     
[55] googledrive_2.0.0   rstudioapi_0.13     assertthat_0.2.1   
[58] rmarkdown_2.9       httr_1.4.2          R6_2.5.0           
[61] git2r_0.29.0        compiler_4.1.0     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
